README.md

```diff
@@ -110,9 +114,89 @@ by adding the above label under `spec.template.labels` in `productpage-v1`, we c
 
 ## Authenticating requests
 
-Now that the we have [configured the service to be managed by 3scale](#routing-service-traffic-through-the-adapter)
+Now that the we have [configured the service to be managed by 3scale](#routing-service-traffic-through-the-adapter) we can decide how requests should be authenticated.
+Currently there are two supported mechanisms:
+1. The API Key authentication pattern
+1. The Application ID authentication pattern
+
+You can read more detailed information about these patterns and their behaviour in the [3scale documentation](https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management/2.4/html/api_authentication/authentication-patterns).
+
```

Comment:
@unleashed I'd like to point these at some upstream docs as opposed to product docs if they exist? Can't seem to find them if they do.

Reply:
I'd like to say that there is a link to something like "upstream docs". I've been pursuing this since we released our software as free software, but have had little to no success at all in moving people to make it happen, up until recently where I got the ball rolling a bit with at least getting upstream software available (even if just as nightly images and not much else being done).

However, at least we got someone working on a new website, and I while could tell you to go link https://3scale.github.io/, if you look deeper you'll see that it is also pointing to the product documentation. :/

Reply:
Ok, will revisit at some stage perhaps :)